src/Microsoft.OpenApi/Extensions/JsonSchemaBuilderExtensions.cs

```diff
@@ -206,6 +206,7 @@ public void Evaluate(EvaluationContext context)
     /// The nullable keyword
     /// </summary>
     [SchemaKeyword(Name)]
+    [SchemaSpecVersion(SpecVersion.Draft202012)]
     public class NullableKeyword : IJsonSchemaKeyword
```

Comment:
Nullable is not a supported keyword in OpenAPI 3.1  It is implemented by assigning an array to the type property and including null as one of the values in the array.